locales/en/translation.json

```
@@ -103,13 +103,13 @@
         "quit": "Quit",
         "reload": "Reload",
         "report": "Report",
-        "select-all": "Select All",
+        "select-all": "Select all",
         "send-feedback": "Send feedback",
         "should-migrate-to-flexible": "One of the new features of this Time to Leave release is a flexible number of entries for the calendar.\n\nThis new view uses a new database format.\nTo use it, you can either migrate your existing database to the new format or make a fresh start.",
         "show-app": "Show App",
         "something-went-wrong": "Something wrong happened :(",
-        "toggle-dev-tools": "Toggle Developer Tools",
-        "ttl-github": "TTL GitHub",
+        "toggle-dev-tools": "Toggle developer tools",
+        "ttl-github": "TTL github",
```

Let's leave GitHub as is, since it is a company name.
```suggestion
        "ttl-github": "TTL GitHub",
```

---

Agreed and committed